rfc0001.bs

```diff
@@ -6,9 +6,9 @@ Copyright © 2017,2019 the Contributors to the WADG0001/Feb2019, published by th
 <pre class='metadata'>
 Title: WADG0001 WebAPI type extension
 H1: Version 0.4.0
-Date: 2019-02-20
+Date: 2020-07-06
 Shortname: wadg0001
-Revision: 2
+Revision: 3
```

Comment:
Please could you add yourself as an editor of this draft, and remove Ivan Goncharov?

Reply:
Great have done @MikeRalphson, appreciate that - thanks!